src/vs/platform/native/electron-main/nativeHostMainService.ts

```diff
@@ -879,6 +883,46 @@ export class NativeHostMainService extends Disposable implements INativeHostMain
 		}
 	}
 
+	async openGPUInfoWindow(windowId: number | undefined, options: { zoomLevel: number }): Promise<void> {
+		if (!this.gpuInfoWindow) {
+			this.gpuInfoParentWindow = BrowserWindow.getFocusedWindow();
+			if (this.gpuInfoParentWindow) {
+				this.gpuInfoWindow = new BrowserWindow({
```

Comment:
Are we maybe able to use the shared window options from here: https://github.com/microsoft/vscode/blob/27651b404ecb97565cb49448a2dbae8fc2e52ea8/src/vs/platform/windows/electron-main/windows.ts#L126